src/main/resources/application.yaml

```diff
@@ -60,7 +60,7 @@ security:
   authorisedServices:
     - "wa_workflow_api"
     - "wa_case_event_handler"
-    - "camunda-bpm"
+    - "camunda-local-bpm"
```

Comment:
careful with this going to master. You can set the local profile instead: 
![Screenshot 2021-02-11 at 12 47 02](https://user-images.githubusercontent.com/3930991/107638265-3e71aa80-6c67-11eb-9dc7-c9126215f8f1.jpg)